frontstage/views/surveys/add_survey_submit.py

```diff
@@ -49,4 +59,22 @@ def add_survey_submit(session):
     logger.info('Successfully retrieved data for confirm add organisation/survey page',
                 case_id=case_id, party_id=party_id)
     return redirect(url_for('surveys_bp.get_survey_list', _anchor=(business_party_id, added_survey_id), _external=True, business_party_id=business_party_id,
-                            survey_id=added_survey_id, tag='todo'))
+                            survey_id=added_survey_id, tag='todo', already_enrolled=already_enrolled))
+
+
+def is_business_enrolled(associations, survey_id, party_id):
+    """
+    Makes two checks as a business can be enrolled  on a survey but not the respondent, which would causes a single
+    check to break if another respondent from the business had previously enrolled on that survey
+
+    :param associations: List of respondents and their enrolled surveys
+    :param survey_id: id of the added survey
+    :param party_id: id of the respondent
+    :return: True if respondent and the business is enrolled on the survey and false otherwise
+    """
+    for info in associations:
```

Comment:
Looking at it, it should be `for association in associations`.  Info is too generic